ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -1591,29 +1591,29 @@ pub(crate) fn const_int_bytes_to_string(num_bits: u32, bytes: &[u8]) -> String {
 #[deku_derive(DekuRead)]
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub(crate) struct IntegerTy {
-    pub(crate) num_bits: u32,
+    bitw: u32,
```

Comment:
May as well add a doc string while we are here.

Reply:
You're right, but I'd rather not do that here. Fish, frying.